packages/stone-phaser/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+pkgname=stone-phaser
+pkgver=0.1.2
+pkgrel=1
+pkgdesc='A classic analog phaser effect LADSPA, LV2 and VST2 plugin'
```

Comment:
Nit: we're sometimes referring to VST2 as just VST (see [midiomatic](https://arch.osamc.de/#x86_64/midiomatic)) and sometimes VST2. Personally, VST2 makes more sense as it's more explicit but then the package group is also called `vst-plugins` (probably for historic reasons)..

Reply:
Yes, I think we should distinguish between VST2 and VST3 at least in the description.

The issue with the group name is sth that bugs me too, we should also discuss this with @dvzrv, Either the groud should be renamed to `vst2-plugins` or it should be documented somewhere that `vst-plugins` refers to VST2 and VST3 plugin packges should use `vst3-plugins`.

Reply:
There is unfortunately no (wiki) documentation on the group names (yet).
vst-plugins refers to VST2 and vst3-plugins refers to VST3 plugins though.

A rename of vst-plugins to vst2-plugins is possible but requires to touch/rebuild a few packages.